RecoLocalFastTime/FTLClusterizer/plugins/BuildFile.xml

```diff
@@ -0,0 +1,12 @@
+<use   name="Geometry/CaloGeometry"/>
```

Comment:
keeping FTL for the moment  to avoid confusion for the moment. In the future we will need to rename everything because eg DataFormats are still kept in FTLRecHit or similarly the RecHit code is in FTLRecHit

Reply:
Yeah in 2019 will start the package migration.